### PR TITLE
[TACHYON-470] Fix TachyonConf.getFloat by removing default value

### PR DESCRIPTION
**Pull request by apc999**
JIRA: https://tachyon.atlassian.net/browse/TACHYON-470

getFloat is still taking a default value. This PR aims to remove the default value parameter.

### CODE REVIEW

File: common/src/main/java/tachyon/conf/TachyonConf.java [hunk 1]
```diff
@@ -278,7 +278,7 @@ public double getDouble(String key) {
     throw new RuntimeException("Invalid configuration key " + key + ".");
   }
 
-  private float getFloat(String key, final float defaultValue) {
+  private float getFloat(String key) {
```
Comment (miasantreble) [inline common/src/main/java/tachyon/conf/TachyonConf.java]: I guess now you can make it public?

Comment (gpang) [inline common/src/main/java/tachyon/conf/TachyonConf.java]: Does this actually ever get called? This is private and I can't find it in this file.

**Comment by gpang** [comment]
I just had a minor question. Thanks @apc999 !

File: common/src/main/java/tachyon/conf/TachyonConf.java (hunk 1 above)
Comment (miasantreble) [inline common/src/main/java/tachyon/conf/TachyonConf.java]: it's not being used anywhere and I turned it private to make sure it's not used until we fix it (drop the default value)

Comment (apc999) [inline common/src/main/java/tachyon/conf/TachyonConf.java]: thanks for the catch, yes it should be `public`.
and yes, it is not really used by other functions yet. But I feel this util method might be good to have.

**Comment by gpang** [comment]
LGTM

**Pull request merged**